theano/gof/type.py

```diff
@@ -997,10 +1019,26 @@ def __init__(self, *args, **kwargs):
             type(self).__name__ + ': expected 0 or only 1 extra parameter "ctype".'
         ctype = kwargs.pop('ctype', 'int')
```

Comment:
I saw elsewhere a pop ctype that default to double. why not always the same dtype as default?